File: TouchPOS_API/Models/RPT_ReceiveModels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
    public class RPT_ReceiveModels
    {
        public class Rceive_data
        {
            public string TR_ID { get; set; }
            public string DOCTYPE { get; set; }
            public string DOCNO { get; set; }
            public string DOCDATE { get; set; }
            public string STCODE { get; set; }
            public string PERCODE { get; set; }
            public string PERNAME { get; set; }
            public string REF_DOCNO { get; set; }
            public Double SEQ { get; set; }
            public string PCODE { get; set; }
            public string PNAME { get; set; }
            public Double qty { get; set; }
            public int IDC { get; set; }
            public string UNITCODE { get; set; }
            public string UNITNAME { get; set; }
            public Double UNIT_COST { get; set; }
            public Double ST_QTY { get; set; }
            public string ST_UNITCODE { get; set; }
            public string ST_UNITNAME { get; set; }
            public Double ST_COST { get; set; }
            public Double TOTAL_COST { get; set; }
            
            public string PGCODE { get; set; }
            public string PGNAME { get; set; }
           
        }

        public class Fillter_Receive
        {
            public string DOCTYPE { get; set; }
            public string DCDATE { get; set; }
            public string DOCNO { get; set; }
            public string REFDOCNO { get; set; }
            public string PERCODE { get; set; }
        }

    }
}

File: Blackoffice/Models/RPT_ReceiveModels.vb
Imports MD

Public Class RPT_ReceiveModels
    Public Class value
        Public Respon As New ResponseModels
        Public Data As List(Of Rceive_data)
    End Class

    Public Class Rceive_data
        Public Property TR_ID As String
        Public Property DOCTYPE As String
        Public Property DOCNO As String
        Private _DOCDATE As String
        Public Property DOCDATE() As String
            Get
                Return _DOCDATE
            End Get
            Set(value As String)
                Dim d1 As DateTime
                d1 = CDate(value)
                _DOCDATE = CL_date.Condate(d1, "dd/MM/yyyy", "TH")
            End Set
        End Property
        Public Property STCODE As String
        Public Property PERCODE As String
        Public Property PERNAME As String
        Public Property REF_DOCNO As String
        Public Property SEQ As Double
        Public Property PCODE As String
        Public Property PNAME As String
        Public Property qty As Double
        Public Property IDC As Integer
        Public Property UNITCODE As String
        Public Property UNITNAME As String
        Public Property ST_QTY As Double
        Public Property ST_UNITCODE As String
        Public Property ST_UNITNAME As String
        Public Property PGCODE As String
        Public Property PGNAME As String

        Public Property UNIT_COST As Double
        Public Property ST_COST As Double
        Public Property TOTAL_COST As Double
    End Class

    Public Class Fillter_Receive
        Public Property DOCTYPE As String = ""
        Public Property DCDATE As String = ""
        Public Property DOCNO As String = ""
        Public Property REFDOCNO As Object = ""
        Public Property PERCODE As Object = ""

    End Class

End Class
